internal/storage/migrations/0008_credential_stage_self_discovery.go

```
@@ -0,0 +1,188 @@
+	return nil
+}
+
+// ValidateRecoverable0008 accepts the pre- and post-migration table shapes so a
+// partially applied rebuild can be resumed safely.
+func ValidateRecoverable0008(db *gorm.DB) error {
+	if !db.Migrator().HasTable(credentialStageTable0008) {
+		return fmt.Errorf("validate recoverable credential stage self discovery: table %q is missing", credentialStageTable0008)
+	}
+	definition, err := credentialStageAuthMethodDefinition0008(db)
+	if err != nil {
+		return nil
```

golangci-lint (2.13.2)

[error] error is not nil (line 128) but it returns nil

(nilerr)